src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java

```diff
@@ -63,7 +67,7 @@ public void testGetIntFromInaccessibleField() throws NoSuchFieldException {
 
     @Test
     public void testTokenValueIncorrect() {
-        final Integer id = Integer.MAX_VALUE - 1;
+        final Integer id = 184;
```

Comment:
please explain this change.
184 will be good value ones we come to extension of tokens on next java syntax extension, that is why we used last possible value.